crates/sui-protocol-config/src/lib.rs

```diff
@@ -2684,6 +2686,7 @@ impl ProtocolConfig {
                     cfg.feature_flags.rethrow_serialization_type_layout_errors = true;
                 }
                 56 => {}
+                57 => {}
```

Comment:
I think you should be fine with just 56? I created it for some small framework changes after the cut on Thursday.